### DISCUSSION TITLE
is it possible to feature better functionality with plotsquared?

### DISCUSSION THREAD

**Discussion by ethanmaul** [GriefPrevention/GriefPrevention#717] [Ideas]
when i use plotsquared on my creative plots world of my server what also has grief prevention installed for my survival world but when i have grief prevention on my creative world it doesn't allow me to build anything on my own claimed plot but when i disable grief prevention on that world and just leave it on my survival world i can break anything including other peoples plots?

**Comment by bigpresh** [comment]
I've never used Plotsquared, but it's not quite clear what's actually happening, what you expect to happen, and what your config actually looks like.

E.g. "i can break anything including other peoples plots?" - GriefPrevention will protect claims, it doesn't know about plotsquared plots.  I'm not sure why you'd expect it to, either - isn't that one of the things Plotsquared does for you anyway?

Maybe if you can provide a bit more detail about what you think it should do vs what it's doing for you it might make more sense, but at the moment it sounds like you're trying to use two different types of land protection plugins, which work differently, and expecting them to seamlessly work together, but it's not clear even how/why.
